From a1a36613bf16f56a7fd3657d4cfd05ae321db1ab Mon Sep 17 00:00:00 2001
From: Thiago Macieira <thiago.macieira@intel.com>
Date: Mon, 8 Apr 2019 12:46:33 -0700
Subject: [PATCH] Build with LLVM 8

Signed-off-by: Thiago Macieira <thiago.macieira@intel.com>
---
 IGC/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/IGC/CMakeLists.txt b/IGC/CMakeLists.txt
index 4831e63..01e418e 100644
--- a/IGC/CMakeLists.txt
+++ b/IGC/CMakeLists.txt
@@ -3295,7 +3295,7 @@ else()
     
     message(STATUS "[IGC] LLVM targets are not defined. Searching for LLVM.")
     if(NOT IGC_PREFERRED_LLVM_VERSION)
-        set(IGC_PREFERRED_LLVM_VERSION "7.0.0")
+        set(IGC_PREFERRED_LLVM_VERSION "8.0.0")
     endif()
     
     # Look for LLVM sources.
-- 
2.21.0

