--- intel-graphics-compiler-igc_release_2019-02-25/visa/LocalScheduler/CMakeLists.txt~	2019-02-22 12:37:30.000000000 +0000
+++ intel-graphics-compiler-igc_release_2019-02-25/visa/LocalScheduler/CMakeLists.txt	2019-03-10 14:04:35.939314992 +0000
@@ -47,7 +47,7 @@
   add_definitions(-D_HAS_EXCEPTIONS=0)
 endif()
 
-add_library(LocalScheduler ${LocalScheduler_SOURCES} ${LocalScheduler_HEADERS})
+add_library(LocalScheduler STATIC ${LocalScheduler_SOURCES} ${LocalScheduler_HEADERS})
 
 if(IGC_BUILD AND MSVC)
 #set up standard defines from the common WDK path.
